udp_mux.go

```diff
@@ -151,9 +134,6 @@ func (m *UDPMuxDefault) Close() error {
 		m.mu.Lock()
 		defer m.mu.Unlock()
 
-		// close udp conn and prevent packets coming in
-		err = m.udpConn.Close()
```

Comment:
I think we need to stop connWorker after Close is called. We shouldn't continue reading from it after it's closed.

Reply:
Fixed!